src/bandersnatch_filter_plugins/filename_name.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import List
+
+from bandersnatch.filter import FilterFilenamePlugin
+
+logger = logging.getLogger("bandersnatch")
+
+_plugin_initialized: bool = False
+_patterns: List[str] = []
+_packagetypes: List[str] = []
+
+
+def _init_once(self):
+    global _plugin_initialized
+
+    if _plugin_initialized:
+        return
+
+    try:
+        for platform in self.configuration["blacklist"]["platforms"].split("\n"):
+
+            if platform.lower() in ("windows", "win"):
+                _patterns.extend([".win32", "-win32", "win_amd64", "win-amd64"])
+                _packagetypes.extend(["bdist_msi", "bdist_wininst"])
+
+            elif platform.lower() in ("macos", "macosx"):
+                _patterns.extend(["macosx_", "macosx-"])
+                _packagetypes.extend(["bdist_dmg"])
+
+            elif platform.lower() in ("freebsd"):
+                _patterns.extend(["freebsd"])
+
+            elif platform.lower() in ("linux"):
+                _patterns.extend(
+                    [
+                        "linux-i686",
+                        "linux-x86_64",
+                        "linux_armv7l",
+                        "linux-armv7l",
+                        "manylinux1_",
```

Comment:
There are many more here I think - Is there any libary with them defined? wheel module?

Reply:
after reading some PEP and setuptools, I think only 'manylinux2010_' and 'linux-armv6l' are missing.
I have added them in my dev branch